src/type/directives.js

```diff
@@ -44,12 +44,17 @@ export class GraphQLDirective {
   locations: Array<DirectiveLocationEnum>;
   args: Array<GraphQLArgument>;
   astNode: ?DirectiveDefinitionNode;
+  repeatable: boolean;
 
   constructor(config: GraphQLDirectiveConfig): void {
     this.name = config.name;
     this.description = config.description;
     this.locations = config.locations;
     this.astNode = config.astNode;
+    this.repeatable =
+      config.repeatable === undefined || config.repeatable === null
```

Comment:
In JS you can do that with a single `config.repeatable == null` and we already using it in couple places.

Reply:
I believe `config.repeatable === undefined` is still necessary since I defined the flow type like this:

```
repeatable?: ?boolean,
```

If I don't do the check, I will get following flow error:

```
Cannot assign `config.repeatable === null ? false : config.repeatable` to `this.repeatable` because:
 - null or undefined [1] is incompatible with boolean [2].
 - undefined [1] is incompatible with boolean [2].
```

Reply:
@OlegIlyenko Note `== null` (two equal signs) compares both to `null` and `undefined` and Flow should handle it without problems.

> If I don't do the check, I will get following flow error:

It's because you use `===` instead of `==`.
It's safe and very common pattern in JS to do `== null`:
https://eslint.org/docs/rules/eqeqeq#smart

Reply:
Ah, I see. Haven't noticed this, thanks for the clarification.

I got an impression that the use of `==` is discouraged in javascript community since it relies of a set of coercion rules that many people find error-prone. This is why I tried to avoid using it. I was not aware of the exceptions described in the eslint documentation though. I will update the code and use `==`.